src/util.ts

```diff
@@ -0,0 +1,87 @@
+import { Logger } from './logger';
+
+const temp = require('temp');
+const fs = require('fs');
+const logger:Logger = Logger.getInstance();
+
+
+export class TempFileHandler {
+    track: any;
+
+    create(TextEditor) {
+        let tempFile = temp.openSync({suffix: '.py'});
```

Comment:
Any check on module imports will have issues with this, see https://github.com/AtomLinter/linter-pylint/issues/57#issuecomment-208030298 for a nice description of the issue here and a possible, although ugly, workaround.